contracts/votingStrategy/IVotingStrategy.sol

```diff
@@ -33,7 +33,7 @@ abstract contract IVotingStrategy {
    * set the round for which the voting contracts is to be used
    *
    */
-  function init() external {
+  function init() external virtual {
```

Comment:
^ do we need to make it virtual ? no harm in having it linked to the round
it's going to be a clone anyways